app/assets/javascripts/components/page/page_comments.es6.jsx

```diff
@@ -6,6 +6,7 @@ class StudentComments extends Component {
   static get propTypes() {
     return {
       comments: React.PropTypes.array.isRequired,
+      type: React.PropTypes.string.isRequired,
```

Comment:
Since this is like an `enum`, let's specify what types to expect.
Example below:
`type: React.PropTypes.oneOf(['login', 'signup']).isRequired,`